packages/deployer/utils/deployments.js

```diff
@@ -97,16 +104,16 @@ function getAllDeploymentFiles(info) {
  * @returns {string} The path of the folder
  */
 function getDeploymentFolder(info) {
-  info = _populateDefaults(info);
-
-  return path.join(info.folder, info.network, info.instance);
+  const { folder, network, instance } = _populateDefaults(info);
+  return path.resolve(folder, network, instance);
 }
 
 function _populateDefaults(info) {
-  return { ...info, ...DeploymentInfo };
```

Comment:
This is a fix, on the old code the default values were overriding the given ones.